addons/fleet/migrations/11.0.0.1/post-migration.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.map_values(
+        env.cr, openupgrade.get_legacy_name('category'), 'category',
+        [('both', 'contract')], table='fleet_service_type',
+    )
+    openupgrade.map_values(
```

Comment:
You have made it the contrary: the values you map are new, not the contrary. The best chance is to execute manually `scheduler_manage_contract_expiration` method of `fleet.vehicle.log.contract` for having the possible contracts on the correct state.

Reply:
Obviously, there's no need of copying the column.